content/tutorials/manage-the-rippled-server/installation/build-run-rippled-ubuntu.md

```diff
@@ -38,7 +34,7 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install `ripple
 
 4. Install CMake.
 
-    Version 1.7.2 of `rippled` requires CMake 3.9.0 or higher. For the purposes of this tutorial, we used CMake 3.13.3.
+    Version 1.8.1 of `rippled` requires CMake 3.9.0 or higher. For the purposes of this tutorial, we used CMake 3.13.3.
```

Comment:
Could we use a variable instead of manually updating the version number?

Reply:
We could, but it's not very helpful since we often need to update these compatibility statements with the release anyway, and it could be misleading if we update the variable without checking all the places it's used. Without a variable, it always provides accurate, if possibly out-of-date, information if we don't update it.